.evergreen/install-mongodb-client-encryption.sh

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env bash
+set +o xtrace # Do not write AWS credentials to stderr
+
+# Initiail checks for running these tests
+if [ -z ${INSTALL_DIR+omitted} ]; then echo "INSTALL_DIR is unset" && exit 1; fi
```

Comment:
Don't mind this. but could we just leave it as `mongodb-client-encryption`? When is `INSTALL_DIR` an alternate name, and why?